src/Settings.php

```diff
@@ -235,6 +235,22 @@ public static function getPathsFromStdIn()
         $lines = explode("\n", rtrim($content));
         return array_map('rtrim', $lines);
     }
+
+    /**
+     * @param string $path
+     * @param string $os
+     * @return string
+     */
+    public static function escapeString(string $path, string $os = PHP_OS)
+    {
+        if (stripos(strtoupper($os), 'WIN') === 0) {
```

Comment:
```suggestion
        if (stripos($os, 'WIN') === 0) {
```

`stripos()` is already a case-insensitive comparison, so no need for the `strtoupper()`.

Reply:
seems the comment is misplaced. the discussion thread was started about not using strtoupper:
- https://github.com/php-parallel-lint/PHP-Parallel-Lint/pull/149/files#r1346640089